app/build.gradle

```diff
@@ -230,16 +231,17 @@ dependencies {
     implementation "com.apollographql.apollo3:apollo-rx2-support:$apolloVersion"
 
     implementation 'com.facebook.android:facebook-android-sdk:16.0.0'
-    implementation("com.google.android.play:review-ktx:2.0.1")
-    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation("com.google.android.play:review-ktx:2.0.2")
+    implementation 'androidx.constraintlayout:constraintlayout:2.2.1'
     implementation 'com.google.android.flexbox:flexbox:3.0.0'
     implementation 'com.google.android.material:material:1.6.1'
```

Comment:
Lates is com.google.android.material:material:1.12.0, does not have dependencies, but many imports break likely better to handle isolated